tests/e2e/CMakeLists.txt

```diff
@@ -1,25 +1,32 @@
 # Set up C++ functions for e2e tests
 function(add_query_module target_name src)
-  add_library(${target_name} SHARED ${src})
```

Comment:
We have to introduce [cmake-format](https://cmake-format.readthedocs.io/en/latest/), because all of our CMake files has different formatting...